app/models/publication.rb

```diff
@@ -144,7 +144,7 @@ def sync_publication_hash_and_db
   end
 
   def update_from_pubmed
-    return false if pmid.blank?
+    return false if pmid.blank? || !pubmed_pub?
 
```

Comment:
just being very cautions ... we really should not be updating from pubmed source unless this was originally sourced from pubmed (this verifies this a pubmed sourced record)